examples/user-notification.amp.html

```diff
@@ -0,0 +1,352 @@
+  <style amp-custom>
+    body {
+      margin: 0;
+      font-family: 'Georgia', Serif;
+    }
+
+    .brand-logo {
+      font-family: 'Open Sans';
+    }
+
+    .ad-container {
+      display: flex;
+      justify-content: center;
+    }
+
+    .content-container p {
+      line-height: 24px;
+    }
+
+    header,
+    .article-body {
+      padding: 15px;
+    }
+
+    .lightbox {
+      background: #222;
+    }
+
+    .full-bleed {
+      margin: 0 -15px;
+    }
+
+    .lightbox-content {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+
+      display: flex;
+      flex-direction: column;
+      flex-wrap: nowrap;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .lightbox-content p {
+      color: #fff;
+      padding: 15px;
+    }
+
+    .lightbox amp-img {
+      width: 100%;
+    }
+
+    figure {
+      margin: 0;
+    }
+
+    figcaption {
+      color: #6f757a;
+      padding: 15px 0;
+      font-size: .9em;
+    }
+
+    .author {
+      display: flex;
+      align-items: center;
+
+      background: #f4f4f4;
+      padding: 0 15px;
+
+      font-size: .8em;
+
+      border: solid #dcdcdc;
+      border-width: 1px 0;
+    }
+
+    .header-time {
+      color: #a8a3ae;
+      font-family: 'Roboto';
+      font-size: 12px;
+    }
+
+    .author p {
+      margin: 5px;
+    }
+
+    .byline {
+      font-family: 'Roboto';
+      display: inline-block;
+    }
+
+    .byline p {
+      line-height: normal;
+    }
+
+    .byline .brand {
+      color: #6f757a;
+    }
+
+    .standfirst {
+      color: #6f757a;
+    }
+
+    .mailto {
+      text-decoration: none;
+    }
+
+    #author-avatar {
+      margin: 10px;
+      border: 5px solid #fff;
+      width: 50px;
+      height: 50px;
+      border-radius: 50%;
+    }
+
+    h1 {
+      margin: 5px 0;
+      font-weight: normal;
+    }
+
+    footer {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      height: 226px;
+      background: #f4f4f4;
+    }
+
+    hr {
+      margin: 0;
+    }
+
+    amp-img {
+      background-color: #f4f4f4;
+    }
+  </style>
+  <script async custom-element="amp-analytics" src="../dist/v0/amp-analytics-0.1.max.js"></script>
+  <script async custom-element="amp-user-notification" src="../dist/v0/amp-user-notification-0.1.max.js"></script>
+  <script async src="/examples/viewer-integr.js"></script>
+  <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <script async src="../dist/amp.js"></script>
+</head>
+<body>
+
+  <amp-analytics type="googleanalytics">
+  <script type="application/json">
+  {
+    "vars": { "account_id": "123456" }
+    "triggers": [{
+      "on": "LOAD",
+      "request": "pageview"
+    }]
+  }
+  </script>
+  </amp-analytics>
+
+  <amp-user-notification
+      layout=container
```

Comment:
No, not quite. Layout system is meant to be used as an automatic layout. The goal is to be able to layout elements even before their implementation is downloaded and instrumented. Once the implementation takes over the element - it can do, more or less, whatever it wants. We still like to keep it close to the original layout, but in some cases it's justifiable to change it drastically. There's no need to change layout attribute or add/remove existing layout classes. If they are needed for some reason - sure. But normally you don't need them. See lightbox implementation: it just force the element visible and positions the element manually - even animates into that position.